README.md

```diff
@@ -4,4 +4,93 @@
 
 This repo will contain standard global configurations for ACCESS-ESM1.5, the ACCESS Coupled Earth System Model.
 
+This is an "omnibus repository": it contains multiple related configurations, and each
+configuration is stored in a separate branch.
+
+Branches utilise a simple naming scheme:
+
+`{release}-{scenario}[+{modifier}]`
+
+where `release` signifies this is the release branch that is tested, versioned and ready for use, `scenario` is the base experimental design with optional `modifiers`. All configurations are assumed to be global extent with nominal 1 degree resolution.
+
+Some examples of possible values of the specifiers:
+
+* scenario: historical, preindustrial, ssp126
+* modifier: concentration, interactiveC, noLUC
+
+where scenario is typically a [CMIP experiment identifier](https://wcrp-cmip.github.io/CMIP6_CVs/docs/CMIP6_experiment_id.html), concentration and interactiveC describe the CO2 cycling protocol, and noLUC is no land-use change.
+
 Most configurations are adapted from work by the [CLEX CMS team](https://github.com/coecms/access-esm).
+
+## Supported configurations
+
+All available configurations are browsable under [the list of branches](https://github.com/ACCESS-NRI/access-esm1.5-configs/branches) and should also be listed below:
+
+| Branch | Configuration Description |
+| ------ | ------------------------- |
```

Comment:
Do we put in what we know will be there, or leave blank and update when they're available?

Reply:
Leave blank and add, I reckon!